perenv.macro.js

```diff
@@ -27,7 +27,7 @@ function loadPerEnvMacro({ state, babel, references }) {
       nullableReference.type === "BooleanLiteral" &&
       nullableReference.value;
 
-    const envMapKeys = envMapReference.properties.map((p) => p.key.name);
+    const envMapKeys = envMapReference.properties.map((p) => p.key.name || p.key.value);
```

Comment:
We're using this same functionality to pull our the value here as on line 40. Would prefer to create a function.

Also, we should probably do a more explicit type check.

something like (I don't love this name)

```js
function expectedEnvValueFromKey(key) {
  if (key. type === 'StringLiteral') {
    return key.value
  }
  if(key.name) {
    return key.name
  }
  throw new Error(`Unsupported key in loadPerEnvMap`);
}